File: tests/week12/total_bill.js
function total_bill(bill_list) {

    var total = 0;
  for (var i = 0; i < bill_list.length; i++) {
    for (let food of bill_list[i]) {
        var price = food.price.replace(/^\$/, '');
        total += Number(price);
    }
  }
    return total;
}

module.exports = total_bill;

File: tests/week12/total_bill.pl
#!/usr/bin/perl -w

$file = "$ARGV[0]";

open FILE, "<", "$file" or die;
@contents = <FILE>;
$counter = 0;
$total = 0;
while ($counter < $#contents + 1) {
    if ($contents[$counter] =~ /\"price\": /) {
        # print "$contents[$counter]\n";
        $contents[$counter] =~ s/\{\"name\": \"[A-Za-z ]+\", \"price\": \"\$//;
        $contents[$counter] =~ s/\"\},?$//;
        $contents[$counter] =~ s/^\s+//;
        #print "$contents[$counter]\n";
        $total += $contents[$counter];
    }
    $counter++;
}
$total = $total;
$number = sprintf("%.2f", "$total");
print "\$"."$number\n";

close FILE;
